dhcpv4/modifiers.go

```diff
@@ -43,6 +43,18 @@ func WithGatewayIP(ip net.IP) Modifier {
 	}
 }
 
+//WithRelayOptions copies the relay options from the request to the reply
+func WithRelayOptions(request *DHCPv4) Modifier {
```

Comment:
`WithRelayAgent`? There's other relay options, right?

Reply:
WithRelayAgentInfo might be best?